server/src/assets/dto/get-asset.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { IsOptional } from 'class-validator';
+
+import { Asset } from '../entities/asset.entity';
+
+export class GetAssetsDto extends PartialType(Asset) {
+  @IsOptional()
```

Comment:
Is the intention of extending from the entity that you can use the entity properties to filter?

Reply:
huh...
yes, i believe so. but also, i was just following the pattern we've been doing. i don't remember why we started doing this, and now that i look, they aren't doing that in the nest docs
so i think creating less coupling to the entity would be better. will remove extensions